AmazonChimeSDKDemo/AmazonChimeSDKDemo/DeviceSelectionViewController.swift

```diff
@@ -27,6 +28,10 @@ class DeviceSelectionViewController: UIViewController {
         videoDevicePicker.dataSource = self
         videoFormatPicker.delegate = self
         videoFormatPicker.dataSource = self
+
+        /// Update the visibility of Audio Device selection based on the audio mode
+        updateAudioState(audioEnabled: true)
```

Comment:
The same comment in Android, are you supposed to expose audio enable/disable option to builders now?

Reply:
Not supposed to. Removed.